scripts/cibuild

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Build project for release.
+"
+}
+
+if [ "${BASH_SOURCE}[0]}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+    ./scripts/setup "$1"
+
```

Comment:
Right now the Jenkins job runs `cibuild` and `cipublish` in succession. I checked a few jobs and it looks that roughly matches what the others do. The `setup` thing was another imitation of ARKF: looks like we call `setup` there then `cibuild`.